### DISCUSSION TITLE
Test does not start

### DISCUSSION THREAD

**Discussion by KadircanBoztepe** [getgauge/gauge#2093] [General]
the gauge scenario is starting and "test started" written on console but it's stuck at this point.  i use below versions.

/opt/homebrew/bin/gauge run --machine-readable --hide-suggestion --simple-console /xxxx
Testing started at 00:53 ...

Process finished with exit code 137 (interrupted by signal 9: SIGKILL)

Gauge version: 1.4.1
html-report (4.1.2)
java (0.7.15)
screenshot (0.1.0)
xml-report (0.2.3)

pom.xml
<dependency>
            <groupId>com.thoughtworks.gauge</groupId>
            <artifactId>gauge-java</artifactId>
            <version>0.7.15</version>
        </dependency>

**Comment by sriv** [comment]
hi - does this happen with a specific project or is this something that you see across gauge projects in your machine?

**Reply by mertmansuroglu** [reply]
Hello, I also faced the same problem while trying to install gauge on my mac. Basically, in my terminal, I put "brew install gauge". Then, installed the gauge install components which in turn , initiated the test from example.spec. However, it's stuck at this point where the author of the issue has mentioned. I am facing this issue in all of my projects with gauge. You can see the ss below regarding the problem. Could you please help me if you have found the solution?
![Screen Shot 2021-09-10 at 14 08 06](https://user-images.githubusercontent.com/63956446/132844566-c76bef28-2766-43fb-b692-84c7f757c03a.png)

**Reply by sriv** [reply]
What version of java are you on @mertmansuroglu ?

Also, are you able to run the tests from your terminal using `gauge run specs` ?

FWIW, I am on 

```
Gauge version: 1.4.1
Plugins
-------
dotnet (0.4.1)
html-report (4.1.2)
inprogress (0.2.0)
java (0.7.15)
js (2.3.17)
python (0.3.17)
screenshot (0.1.0)
ts (0.1.0)
xml-report (0.2.3)
```

And I am using OpenJDK 14.

Running `gauge init java` and then `gauge run specs` works fine for me, so I am still trying to guess what is different in yours (and also @KadircanBoztepe 's) projects/environments.

**Comment by mertmansuroglu** [comment]
<img width="559" alt="Screen Shot 2021-09-16 at 15 59 21" src="https://user-images.githubusercontent.com/63956446/133616419-7110dba8-35b9-4eb6-b501-ea018f28db5a.png">
<img width="516" alt="Screen Shot 2021-09-16 at 15 40 47" src="https://user-images.githubusercontent.com/63956446/133613725-d639eba9-5ed9-44cc-9367-703ec5d9b0a8.png">
Hello, here i have seen this screen after running the command "gauge run specs"
Also, I have added the required info you have asked

**Reply by sriv** [reply]
hi, it seems that your project uses selenium, and if  you have initialized a maven(or gradle) based template, you need to use those tools to run the project.

For ex: if you ran `gauge init java_maven_selenium` then the tests need to be run using `mvn test` or `mvn test-compile gauge:execute`

**Comment by mertmansuroglu** [comment]
**Hello there, we again faced a similar issue regarding stuck while running the spec file.** **When we tried by setting the timeout to 10000 ms, the following error came "Failed to start gauge API: Timed out connecting to java"** Basically, in the same module, API specs can run perfectly whereas Mobile specs showed the same problem mentioned above. Also, we tried with different java versions to fix the issue but still no luck. You can have a look at the log file below, gauge version with plugin and pom file. By the way, In our project, we are getting use of multipom mechanism, that is a parent pom is set which is covering three different poms called api, mobile and web.


**Note: Another point we want mention is, we cannot debug from spec file using intellij which says connection refused in the console.**


[gauge.log](https://github.com/getgauge/gauge/files/8002559/gauge.log) gauge logs here

<img width="676" alt="Screen Shot 2022-02-04 at 12 51 37" src="https://user-images.githubusercontent.com/63956446/152507849-9826acc8-db79-4cf8-b0fb-901d18075c2a.png">

<img width="588" alt="Screen Shot 2022-02-04 at 12 53 15" src="https://user-images.githubusercontent.com/63956446/152508081-4bfd7a06-4ab5-4bab-b0a4-438a6de07baa.png">

<img width="613" alt="Screen Shot 2022-02-04 at 12 53 35" src="https://user-images.githubusercontent.com/63956446/152508120-fe7d6cdf-7087-4fe3-9be1-c2acf247fea8.png">

<img width="889" alt="Screen Shot 2022-02-04 at 12 56 28" src="https://user-images.githubusercontent.com/63956446/152508560-13a07a63-6bc4-4041-8566-8bbddbe5c88c.png">

<img width="814" alt="Screen Shot 2022-02-04 at 12 58 44" src="https://user-images.githubusercontent.com/63956446/152508923-fff1cdb6-4213-4c98-8e27-7978da588422.png">
